x-pack/platform/plugins/shared/lens/public/app_plugin/shared/edit_on_the_fly/helpers.ts

```diff
@@ -49,7 +48,7 @@ const getDSLFilter = (queryService: DataPublicPluginStart['query'], timeFieldNam
 export const getGridAttrs = async (
   query: AggregateQuery,
   adHocDataViews: DataViewSpec[],
-  deps: LensPluginStartDependencies,
+  data: DataPublicPluginStart,
```

Comment:
only `deps.data` and `deps.dataViews` were used and dataViews are also available on `deps.data.dataViews`